### PR TITLE
Kick SubCommand

### PR DESCRIPTION
**Pull request by HimmelKreis4865**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
This PR adds /p kick  to kick players from your plots.

## Optional information
I just translated it for eng + deu.

### CODE REVIEW

File: resources/eng.ini [hunk 1]
```diff
@@ -229,4 +229,15 @@ clone.desc=clones a plot to another location
...
 clone.wrongid="The plot id should be in the format X;Z"
 clone.nomoney=You do not have enough money to clone this plot
-clone.success=Plot {%0} has been successfully cloned to plot {%1}
\ No newline at end of file
+clone.success=Plot {%0} has been successfully cloned to plot {%1}
+
+; the /p kick Subcommand
+kick.name=kick
+kick.desc=Kick a player from your plot
+kick.usage=/p kick <target: player>
```
Comment (jasonwynn10) [inline resources/eng.ini:237]: Reversed usage. Target usage message arguments should be as `<player: target>`

File: src/MyPlot/EventListener.php [hunk 2]
```diff
@@ -1,356 +0,0 @@
-<?php
```
Comment (jasonwynn10) [inline src/MyPlot/EventListener.php:1]: The Event Listener class is necessary for the plugin to run. What are you doing?

File: src/MyPlot/subcommand/KickSubCommand.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            $sender->sendMessage(TextFormat::RED . $this->translateString("kick.notInPlot"));
+            return true;
+        }
+        if ($target->hasPermission("myplot.admin.kick.bypass")) {
+            $sender->sendMessage(TextFormat::RED . $this->translateString("kick.cannotkick"));
+            $target->sendMessage($this->translateString("kick.attemptkick", [$target->getName()]));
+            return true;
+        }
+        $vector = $this->getPlugin()->getPlotPosition($plot)->add(0, 2, 0)->subtract(1, 0, 1);
+        $pos = new Position($vector->x, $vector->y, $vector->z, $target->getLevel());
```
Comment (jasonwynn10) [inline src/MyPlot/subcommand/KickSubCommand.php:53]: These lines can be simplified

File: src/MyPlot/EventListener.php (hunk 2 above)
Comment (HimmelKreis4865) [reply src/MyPlot/EventListener.php:1]: I added EventListener in the end.
I needed to renew it, what wasn't that easy on mobile.

File: src/MyPlot/subcommand/KickSubCommand.php (hunk 3 above)
Comment (HimmelKreis4865) [reply src/MyPlot/subcommand/KickSubCommand.php:53]: Yea I'll do that now. Thanks. :3

**Comment by HimmelKreis4865** [comment]
I updated everything you wanted from me. :3

File: resources/eng.ini [hunk 4]
```diff
@@ -229,4 +229,15 @@ clone.desc=clones a plot to another location
...
+; the /p kick Subcommand
+kick.name=kick
+kick.desc=Kick a player from your plot
+kick.usage=/p kick <player: target>
+kick.noPlayer=The given player was not found
+kick.notInPlot=The given player is not in your plot
+kick.cannotkick=You cannot kick this player
+kick.attemptkick=Player {%0} attempt to kick you
+kick.success1=You successfully kicked {%0} from plot {%1}
+kick.success2=Player {%0} kicked you from plot {%1}
```
Comment (jasonwynn10) [inline resources/eng.ini:243]: Line Endings must be LF formatted, not CRLF

File: resources/deu.ini [hunk 5]
```diff
@@ -227,4 +227,15 @@ clone.desc=Klont ein Grundstück an eine andere Position
...
+; the /p kick Subcommand
+kick.name=kick
+kick.desc=Kicke einen Spieler von deinem Plot
+kick.usage=/p kick <player: target>
+kick.noPlayer=Der angegebene Spieler konnte nicht gefunden werden
+kick.notInPlot=Der angegebene Spieler befindet sich nicht in deinem Plot
+kick.cannotkick=Du kannst diesen Spieler nicht kicken
+kick.attemptkick=Der Spieler {%0} hat versucht, dich von seinem plot zu kicken
+kick.success1=Du hast erfolgreich {%0} von dem Plot {%1} gekickt
+kick.success2=Der Spieler {%0} hat dich von dem Plot {%1} gekickt
```
Comment (jasonwynn10) [inline resources/deu.ini:241]: Line Endings must be LF formatted, not CRLF

File: src/MyPlot/Commands.php [hunk 6]
```diff
@@ -236,4 +238,4 @@ public function execute(CommandSender $sender, string $alias, array $args) : boo
 		}
 		return true;
 	}
-}
\ No newline at end of file
+}
```
Comment (jasonwynn10) [inline src/MyPlot/Commands.php:241]: Line Endings must be LF formatted, not CRLF

File: src/MyPlot/subcommand/KickSubCommand.php [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        }
+        if ($target->teleport($this->getPlugin()->getPlotPosition($plot)->add(0, 2, 0)->subtract(1, 0, 1))) {
+            $sender->sendMessage($this->translateString("kick.success1", [$target->getName(), $plot->__toString()]));
+            $target->sendMessage($this->translateString("kick.success2", [$sender->getName(), $plot->__toString()]));
+            return true;
+        }
+        $sender->sendMessage($this->translateString("error"));
+        return true;
+    }
+}
```
Comment (jasonwynn10) [inline src/MyPlot/subcommand/KickSubCommand.php:60]: Line Endings must be LF formatted, not CRLF

File: src/MyPlot/subcommand/KickSubCommand.php [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+        if (!$this->getPlugin()->getPlotByPosition($target)->isSame($plot)) {
+            $sender->sendMessage(TextFormat::RED . $this->translateString("kick.notInPlot"));
+            return true;
+        }
+        if ($target->hasPermission("myplot.admin.kick.bypass")) {
+            $sender->sendMessage(TextFormat::RED . $this->translateString("kick.cannotkick"));
+            $target->sendMessage($this->translateString("kick.attemptkick", [$target->getName()]));
+            return true;
+        }
+        if ($target->teleport($this->getPlugin()->getPlotPosition($plot)->add(0, 2, 0)->subtract(1, 0, 1))) {
```
Comment (jasonwynn10) [inline src/MyPlot/subcommand/KickSubCommand.php:52]: This can still be simplified more
```php
$this->getPlugin()->getPlotPosition($plot)->add(-1, 2, -1)
```

File: src/MyPlot/EventListener.php [hunk 9]
```diff
@@ -112,7 +112,7 @@ public function onBlockBreak(BlockBreakEvent $event) : void {
 	 * @param PlayerInteractEvent $event
 	 */
 	public function onPlayerInteract(PlayerInteractEvent $event) : void {
-		if(($event->getAction() === PlayerInteractEvent::RIGHT_CLICK_BLOCK or $event->getAction() === PlayerInteractEvent::RIGHT_CLICK_AIR) and !$event->getItem() instanceof Food)
+		if($event->getAction() === PlayerInteractEvent::RIGHT_CLICK_AIR and $event->getItem() instanceof Food)
```
Comment (jasonwynn10) [inline src/MyPlot/EventListener.php:115]: This change is not related to the content of this PR in any way.

File: src/MyPlot/EventListener.php [hunk 10]
```diff
@@ -256,7 +256,7 @@ public function onBlockSpread(BlockSpreadEvent $event) : void {
 		if(!$this->plugin->isLevelLoaded($levelName))
 			return;
 		$settings = $this->plugin->getLevelSettings($levelName);
-		if(!$settings->updatePlotLiquids) {
+		if(!$settings->updatePlotLiquids and ($this->plugin->getPlotByPosition($event->getBlock()) instanceof Plot or $this->plugin->getPlotByPosition($event->getSource()) instanceof Plot or $this->plugin->isPositionBorderingPlot($event->getBlock()) or $this->plugin->isPositionBorderingPlot($event->getSource()))) {
```
Comment (jasonwynn10) [inline src/MyPlot/EventListener.php:259]: This change is not related to the content of this PR in any way.
